stripe/src/main/java/com/stripe/android/model/ShippingMethod.java

```diff
@@ -120,6 +123,33 @@ public JSONObject toJson() {
         return map;
     }
 
+
+    @Nullable
+    public static ShippingMethod fromString(@Nullable String jsonString) {
+        try {
+            return fromJson(new JSONObject(jsonString));
+        } catch (JSONException ignored) {
+            return null;
+        }
+    }
+
+    public static ShippingMethod fromJson(@Nullable JSONObject jsonObject) {
+        if (jsonObject == null) {
+            return null;
+        }
+        Long amount = optLong(jsonObject, FIELD_AMOUNT);
+        String label = optString(jsonObject, FIELD_LABEL);
+        String detail = optString(jsonObject, FIELD_DETAIL);
+        String identifier = optString(jsonObject, FIELD_IDENTIFIER);
+        String currencyCode = optString(jsonObject, FIELD_CURRENCY_CODE);
+        return new ShippingMethod(
```

Comment:
I don't think this is the right class for `PaymentIntent.shipping`. https://stripe.com/docs/api#payment_intent_object-shipping has `address` (`city`/`country`/`line1`/`line2`/`postal_code`/`state`), `carrier`, `name`, `phone`, and `tracking_number`